CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## New Features
 - PR #185 - Add function to translate PyCUDA gpuarray to CuPy ndarray
 - PR #195 - Add explicit FIR filter functionality
+- PR #197 - Added Ampere support
```

Comment:
PR number in file does not match the actual PR number

Reply:
Thanks! That fixed the CI issue I believe.